src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -12,11 +12,7 @@ def __init__(self):
 
 
 class PimFileTransferService:
-    """Instantiates a file uploader and downloader to have a seamless file reading /
-    writing in the cloud particularly in Ansys lab . Here we are exposing upload and
-    download methods on session objects. These would be no- ops if PyPIM is not
-    configured or not authorized with the appropriate service. This will be used for
-    internal purpose only.
+    """Provides a file transfer service based on PyPIM and simple_upload_server.
```

Comment:
Descriptions for classes should avoid code entities in favor of plain English. If a code entity must be used, you must tag it in doube backticks and indicate what the data type is. For example if this is a method, it should be formatted like this: ... and the ``simple_upload_server()`` method.

Reply:
Done. Thank you.